README.md

```diff
@@ -5,6 +5,15 @@ This test suite will assume that the endpoint exists and that security tokens/co
 
 The test framework will provide a flexible host configuration that can be extended to support both cloud and local platform authentication.
 
+# Installation
+There are a small number of library dependencies. To install them, run the following command within the top level directory of the repository:
+    pip install -r requrements.txt
+
+Once this is done, you'll need to then install the program itself:
+    python setup.py develop
```

Comment:
Either of the followings may be able to replace these lines:

- (prd) `pip install .`
- (dev) `pip install -e .`